src/Digdir.Domain.Dialogporten.Application/Common/ICompactJwsGenerator.cs

```diff
@@ -0,0 +1,147 @@
+using System.Buffers.Text;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.Extensions.Options;
+using NSec.Cryptography;
+
+namespace Digdir.Domain.Dialogporten.Application.Common;
+
+public interface ICompactJwsGenerator
+{
+    string GetCompactJws(Dictionary<string, object?> claims);
+    bool VerifyCompactJws(string compactJws);
+}
+
+public class Ed25519Generator : ICompactJwsGenerator
```

Comment:
Er dette en infrastruktur greie?

Reply:
Nei, er det det? Jeg tenker at det at applikasjonen genererer tokens er forretningslogisk og ikke eksternt for applikasjonen. Alt annet vi har i Infrastructure er knyttet til konkret ekstern infrastruktur (db, events, autorisasjon, registre); dette er jo ikke det?

Reply:
Lar den ligge som del av applikasjonen pt.